bower.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "KaTeX",
-  "version": "0.8.0-pre",
+  "name": "katex",
   "main": [
-    "dist/katex.min.js",
-    "dist/katex.min.css"
+    "dist/katex.js",
+    "dist/katex.css"
```

Comment:
Yeah, I'm not really sure what `main` means I guess. It does say that you should use unminified things though, so at least this change is correct.